File: rebus-rebuscrus/src/java/it/csi/rebus/rebuscrus/vo/ComboVO.java
/*
* SPDX-FileCopyrightText: (C) Copyright 2022 Regione Piemonte
* SPDX-License-Identifier: EUPL-1.2
*/
package it.csi.rebus.rebuscrus.vo;

import java.util.List;

public class ComboVO extends ParentVO {

	private static final long serialVersionUID = 1L;

	
	private String   identificativo;
	private List<DescrizioneVO> valori;

	public String getIdentificativo() {
		return identificativo;
	}

	public void setIdentificativo(String identificativo) {
		this.identificativo = identificativo;
	}

	public List<DescrizioneVO> getValori() {
		return valori;
	}

	public void setValori(List<DescrizioneVO> valori) {
		this.valori = valori;
	}

}

File: rebus-rebuscruswcl/src/app/vo/combo-vo.ts
/*************************************************
Copyright Regione Piemonte - 2022
SPDX-License-Identifier: EUPL-1.2-or-later
***************************************************/

/*
 * @Author: Massimo.durando 
 * @Date: 2018-01-12 10:22:42  
 **/
export class ComboVO {

    constructor(
        public identificativo: TipoCombo,
        public valori: DescrizioneVO[]
    ) { }
}


export class DescrizioneVO {

    constructor(
        public codice: number,
        public descrizione: string
    ) { }
}

export enum TipoCombo {
    CLASSE_VEICOLO = "CLASSE_VEICOLO",
    TIPO_EURO = "TIPO_EURO",
    TIPO_ALIMENTAZIONE = "TIPO_ALIMENTAZIONE",
    TIPO_DISPOSITIVI_PREVENZIONE = "TIPO_DISPOSITIVI_PREVENZIONE",
    TIPO_AUDIO = "TIPO_AUDIO",
    TIPO_VIDEO = "TIPO_VIDEO",
    TIPO_ALLESTIMENTO = "TIPO_ALLESTIMENTO",
    TIPO_FACILITAZIONI = "TIPO_FACILITAZIONI",
    TIPO_IMMATRICOLAZIONE = "TIPO_IMMATRICOLAZIONE",
    PROPRIETA_LEASING = "PROPRIETA_LEASING",
    CATEGORIA_VEICOLO= "CATEGORIA_VEICOLO"


}

